### ISSUE TITLE
[BUG] sequences , combinations and special keys beyond 'enter' or 'up' don't work

### ISSUE THREAD

**Issue by AlbertoEAF** [greena13/react-hotkeys#171]
Basic reproducible example with minimal react App.js (use react-create-app) + npm i -s react-hotkeys:
```javascript
import React from 'react';
import { HotKeys } from 'react-hotkeys';

export default function App(props) {
  let keyMap = {
    act: "shift", // doesn't trigger
    up: "up+down+left", // triggers on the first up
    enter: ["enter"], // correctly triggers
    alt_enter: "alt+enter", // doesn't trigger
  }

  let keyHandlers = {
      act: event => alert(event),
      enter: event => alert("enter!"),
      up: event => alert("up"),
      alt_enter: ev => alert("alt_enter"),
  }

  return (
    <HotKeys keymap={keyMap} handlers={keyHandlers}>
      <div style={{width:"90%", height:"100px", border:"2px solid #333"}}>
        <p>placeholder box as target for HotKeys</p>
      </div>
    </HotKeys>
  )
}
```

**Comment by greena13** [comment]
Hey @AlbertoEAF, this sounds a little similar to some issues that have (hopefully) been resolved in [v2.0.0-pre7](https://github.com/greena13/react-hotkeys/releases/tag/v2.0.0-pre7). Are you able to test and confirm whether you're still experiencing issues?

**Comment by greena13** [comment]
I'm closing this issue for now due to inactivity.

There have been a number of fixes in recent versions that may have solved this. Please let me know if this is not the case, complete with the version of `react-hotkeys`, `react`, the browser version you're using and a `react-hotkeys` log of the failure.
